fw/http_parser.c

```diff
@@ -2264,6 +2272,8 @@ __req_parse_cache_control(TfwHttpReq *req, unsigned char *data, size_t len)
 	}
 
 	__FSM_STATE(Req_I_CC_m) {
+		if (!parser->cache_flags.flag_allowed)
```

Comment:
"flag_allowed == false" means mean there was no comma or semicolumn before this token. However, I agree the purpose of this field should be better documented.

Reply:
OK, got it. FSM should not look backward. If it looks for comma, then it must skip LWSes and when it seems something, which isn't "," or LWS, it should block it since there is no such state. This is just a different processing logic.